salt/modules/dracr.py

```diff
@@ -151,6 +151,47 @@ def __execute_ret(command, host=None,
     return cmd
 
 
+def get_property(host=None, admin_username=None, admin_password=None, property=None):
+    '''
+    .. versionadded:: 2018.3.2
```

Comment:
This version should be `Fluorine` instead.

Can you also document the various options (host, admin_username, etc) and provide a CLI Example? I think there are examples of how this should look in other functions in this file.